### PR TITLE
[DOCFIX] Improve Hive and Presto docs

### PR DESCRIPTION
**Pull request by apc999**
Improve the doc for Hive and Presto according to the doc guide

### CODE REVIEW

File: docs/en/compute/Compute-Presto.md [hunk 1]
```diff
@@ -9,8 +9,6 @@ priority: 2
 * Table of Contents
 {:toc}
 
-## Overview
```
Comment (LuQQiu) [inline docs/en/compute/Compute-Presto.md]: Name it as `Prerequisites`?

File: docs/en/compute/Compute-Presto.md [hunk 2]
```diff
@@ -27,8 +25,9 @@ latency especially when data is remote or network is slow or congested.
 * [Setup Java](Java-Setup.html) for Java 8 Update 60 or higher (8u60+), 64-bit.
```
Comment (LuQQiu) [inline docs/en/compute/Compute-Presto.md]: `[Setup Java](Java-Setup.html) for Java 8 Update 60 or higher (8u60+), 64-bit.` ->  `Setup Java for Java 8 Update 60 or higher (8u60+), 64-bit.`

File: docs/en/compute/Compute-Hive.md [hunk 3]
```diff
@@ -14,51 +14,50 @@ that you can easily store Hive tables in Alluxio's tiered storage.
...
 
-* [Setup Java](Java-Setup.html) for Java 8 Update 60 or higher (8u60+), 64-bit.
+* Setup Java for Java 8 Update 60 or higher (8u60+), 64-bit.
 * [Download and setup Hive](https://cwiki.apache.org/confluence/display/Hive/GettingStarted).
 * An Alluxio cluster has been set up and is running according to either
-[Local Mode](Running-Alluxio-Locally.html) or [Cluster Mode](Running-Alluxio-on-a-Cluster.html).
+[Local Mode]({{ site.baseurl }}{% link en/deploy/Running-Alluxio-Locally.md %}) or
+[Cluster Mode]({{ site.baseurl }}{% link en/deploy/Running-Alluxio-On-a-Cluster.md %}).
 * Make sure that the Alluxio client jar is available.
   This Alluxio client jar file can be found at `{{site.ALLUXIO_CLIENT_JAR_PATH}}` in the tarball
```
Comment (LuQQiu) [inline docs/en/compute/Compute-Hive.md]: Sync the indentation?
The other lines have no space in front of the line.

File: docs/en/compute/Compute-Hive.md [hunk 4]
```diff
@@ -14,51 +14,50 @@ that you can easily store Hive tables in Alluxio's tiered storage.
...
   downloaded from Alluxio [download page](http://www.alluxio.org/download).
   Alternatively, advanced users can compile this client jar from the source code
   by following the [instructions](Building-Alluxio-From-Source.html).
-
-To run Hive on Hadoop MapReduce, please also follow the instructions in
-[running MapReduce on Alluxio](Running-Hadoop-MapReduce-on-Alluxio.html) to make sure Hadoop
-MapReduce can run with Alluxio.
+* To run Hive on Hadoop MapReduce, please also follow the instructions in
+[running MapReduce on Alluxio]({{ site.baseurl }}{% link en/compute/Compute-Hadoop-MapReduce.md %})
+to make sure Hadoop
```
Comment (LuQQiu) [inline docs/en/compute/Compute-Hive.md]: Reduce one line?
```
to make sure Hadoop
MapReduce can work with Alluxio.
```
to 
```
to make sure Hadoop MapReduce can work with Alluxio.
```

File: docs/en/compute/Compute-Hive.md [hunk 5]
```diff
@@ -14,51 +14,50 @@ that you can easily store Hive tables in Alluxio's tiered storage.
...
+MapReduce can work with Alluxio.
+In the following sections of this documentation, Hive is running on Hadoop MapReduce.
 
 ## Basic Setup
 
-Include Alluxio client jar to ensure that Hive can query and access data stored on Alluxio.
+Distribute Alluxio client jar on all Hive nodes and include the Alluxio client jar to Hive
+classpath
+so Hive can query and access data on Alluxio.
```
Comment (LuQQiu) [inline docs/en/compute/Compute-Hive.md]: ditto

File: docs/en/compute/Compute-Hive.md [hunk 6]
```diff
@@ -182,13 +195,80 @@ Instructions and examples till here illustrate how to use Alluxio as one of the
...
+### Add Additional Alluxio client-side properties
+
+There are two ways to specify any Alluxio client properties for Hive queries when
+connecting to Alluxio service:
+
+- Specify the Alluxio client properties in `alluxio-site.properties` and
+ensure that this file is on the classpath of Hive service on each node.
+- Add the Alluxio site properties to `conf/hive-site.xml` configuration file on each node.
+
+#### Example: connect to Alluxio with HA
```
Comment (LuQQiu) [inline docs/en/compute/Compute-Hive.md]: The `Example` is confusing here and in the later section for me when first viewing the doc.
Could we remove the `Example` or using other names?

File: docs/en/compute/Compute-Hive.md (hunk 5 above)
Comment (apc999) [reply docs/en/compute/Compute-Hive.md]: done

File: docs/en/compute/Compute-Hive.md (hunk 6 above)
Comment (apc999) [reply docs/en/compute/Compute-Hive.md]: ok pulled this to a separate section

File: docs/en/compute/Compute-Presto.md (hunk 1 above)
Comment (apc999) [reply docs/en/compute/Compute-Presto.md]: see line 25 for `Prerequisites` . this is plain intro

File: docs/en/compute/Compute-Presto.md (hunk 2 above)
Comment (apc999) [reply docs/en/compute/Compute-Presto.md]: fixed

**Comment by apc999** [comment]
@LuQQiu PTAL

File: docs/en/compute/Compute-Hive.md [hunk 7]
```diff
@@ -182,13 +193,81 @@ Instructions and examples till here illustrate how to use Alluxio as one of the
...
+Alternatively, modify `conf/hive-site.xml` to have:
+
+```xml
+<property>
+  <name>alluxio.user.file.writetype.default</name>
+  <value>CACHE_THROUGH</value>
+</property>
+```
+
+You can use `-h` to display helpful information about the command.
```
Comment (LuQQiu) [inline docs/en/compute/Compute-Hive.md]: This is related to the `alluxio-checker.sh` and should be put in the end of this file

File: docs/en/compute/Compute-Hive.md [hunk 8]
```diff
@@ -182,13 +193,81 @@ Instructions and examples till here illustrate how to use Alluxio as one of the
...
+This command will report potential problems that might prevent you from running Hive on Alluxio.
+
+### Connect to Alluxio with HA
+
+> Tips：after Alluxio 1.8 (exclusive), there is an easier way to configure Hive to connect to Alluxio
+ in fault tolerant mode with Zookeeper. Please follow the instructions in [HDFS API to connect to Alluxio with high availability](Running-Alluxio-on-a-Cluster.html#hdfs-api).
+
+If you are running Alluxio in fault tolerant mode with a Zookeeper service running at
+`zkHost1:2181`, `zkHost2:2181` and `zkHost3:2181`, it requires setting 
+set the Alluxio propertie `alluxio.zookeeper.enabled` and `alluxio.zookeeper.address`
```
Comment (LuQQiu) [inline docs/en/compute/Compute-Hive.md]: propertie -> properties

File: docs/en/compute/Compute-Hive.md [hunk 9]
```diff
@@ -14,51 +14,48 @@ that you can easily store Hive tables in Alluxio's tiered storage.
...
 * [Download and setup Hive](https://cwiki.apache.org/confluence/display/Hive/GettingStarted).
 * An Alluxio cluster has been set up and is running according to either
-[Local Mode](Running-Alluxio-Locally.html) or [Cluster Mode](Running-Alluxio-on-a-Cluster.html).
+  [Local Mode]({{ site.baseurl }}{% link en/deploy/Running-Alluxio-Locally.md %}) or
+  [Cluster Mode]({{ site.baseurl }}{% link en/deploy/Running-Alluxio-On-a-Cluster.md %}).
 * Make sure that the Alluxio client jar is available.
   This Alluxio client jar file can be found at `{{site.ALLUXIO_CLIENT_JAR_PATH}}` in the tarball
   downloaded from Alluxio [download page](http://www.alluxio.org/download).
   Alternatively, advanced users can compile this client jar from the source code
   by following the [instructions](Building-Alluxio-From-Source.html).
```
Comment (LuQQiu) [inline docs/en/compute/Compute-Hive.md]: [instructions](Building-Alluxio-From-Source.html). -> `[instructions]({{ site.baseurl }}{% link en/contributor/Building-Alluxio-From-Source.md %})`

File: docs/en/compute/Compute-Hive.md (hunk 7 above)
Comment (apc999) [reply docs/en/compute/Compute-Hive.md]: oops. good catch

**Pull request merged**